### ISSUE TITLE
Search Suggestions Stop Showing in Start Menu

### ISSUE THREAD

**Issue by munawarb** [nvaccess/nvda#9371]
### Steps to reproduce:

Happens randomly.

### Actual behavior:

When I press the WINDOWS key and start typing the name of an application, NVDA should announce the list of suggestions. Instead, NVDA remains quiet, and I find that the search suggestions are not appearing at all. It seems to happen most often when I'm doing a lot of work in Firefox or Office Applications. I have to resort to pressing WIN+R in these cases and typing a search there, like `firefox` to launch Firefox because the start menu is inoperable. Also, tabbing around the start menu yields no panes for tiles or anything. It's like the start menu has been ghosted; this is the best way to describe the problem. It appears totally blank to me.

### Expected behavior:

NVDA should not crash the start menu, or whatever it's doing, to cause this problem.

**Comment by josephsl** [comment]
Hi, I do know of some folks experiencing this, and possibly traced to confused UIA client implementation in NVDA. As for Windows 10, the developers would like to know exactly which release (there are seven of them now). Thanks.

**Comment by ABuffEr** [comment]
Hi @munawarb ,
are you sure this is a NVDA issue?
In my experience, it's a problem related to Windows, and specifically to Microsoft Text Services Framework, that is, the famous and mysterious "taskhostw.exe {222A245B-E637-4AE9-A93F-A59CA119A75E}" in task manager, often cited in forums without any solution.
I created months ago a .bat to automatically/manually restart this service (mainly for memory problems), and start search functionality always works again after I run it:
```
@echo off
start /b /w schtasks /end /tn "\Microsoft\Windows\TextServicesFramework\MsCtfMonitor"
sleep 5
schtasks /run /tn "\Microsoft\Windows\TextServicesFramework\MsCtfMonitor"
exit /b
```
If you have errors on sleep 5 (it's probably a Cygwin utility), replace it with
`ping -n 5 -w 1000 127.0.0.1>nul 2>&1`

**Comment by munawarb** [comment]
Hi @josephsl, it's build 1809 version 17763.316 Home.

**Comment by ABuffEr** [comment]
Hi @munawarb ,
did you try my solution?
If you agree, I suggest to close this issue, as probably unrelated to NVDA.

**Comment by munawarb** [comment]
Hi. Yes you can go ahead and close it. I haven't had the issue since posting it but I've saved your script in case it does happen again.

**Comment by ABuffEr** [comment]
Ok. You or @Adriani90 can close this issue, then.
